### DISCUSSION TITLE
Request to change repository urls on two apps

### DISCUSSION THREAD

**Discussion by ghost** [codemagic-ci-cd/codemagic-docs#2070] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

First app:
New repository: https://codemagic.io/app/650dc448aabe8b0f34a53f80
Old repository: https://codemagic.io/app/5eb9a0f1a6d15e63cbb05df9

Second app:
New repository: https://codemagic.io/app/650dc5319aef7626f53c4781
Old repository: https://codemagic.io/app/5eb9902aa6d15e6b6c37667d

### Expected results

I would like to preserve the app from the old repository and just update the urls to the new location.

### Actual results

Can't change the repository urls in the ui, I read that I should email support but I am not a paying customer.  Presumably I can still request someone to change this without paying?

**Comment by rafeysup93** [comment]
Hi @VinceERS,

Please share your Personal Account/ Team ID for the verification.

**Comment by ghost** [comment]
Hi, my team id is this I believe: https://codemagic.io/teams/5eb98c1ba6d15eb33331b420

**Reply by rafeysup93** [reply, marked as answer]
Your applications have been moved successfully.
